src/Jackett/Indexers/nCore.cs

```diff
@@ -0,0 +1,214 @@
+﻿using CsQuery;
+using Jackett.Models;
+using Jackett.Services;
+using Jackett.Utils;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NLog;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Jackett.Indexers
+{
+    public class nCore : BaseIndexer, IIndexer
+    {
+        private readonly string SearchUrl = "http://ncore.cc/torrents.php";
+        private static string LoginUrl = "https://ncore.cc/login.php";
+        private readonly string LoggedInUrl = "http://ncore.cc/index.php";
+        private const int MAXPAGES = 3;
+
+        private readonly string enSearch = "torrents.php?oldal={0}&tipus=kivalasztottak_kozott&kivalasztott_tipus=xvidser,dvdser,hdser&mire={1}&miben=name";
+        private readonly string hunSearch = "torrents.php?oldal={0}&tipus=kivalasztottak_kozott&kivalasztott_tipus=xvidser_hun,dvdser_hun,hdser_hun,mire={1}&miben=name";
+        private readonly string enHunSearch = "torrents.php?oldal={0}&tipus=kivalasztottak_kozott&kivalasztott_tipus=xvidser_hun,xvidser,dvdser_hun,dvdser,hdser_hun,hdser&mire={1}&miben=name";
+
+        CookieContainer cookies;
+        HttpClientHandler handler;
+        HttpClient client;
```

Comment:
Don't use httpclient directly, use the iwebclient on the baseindexer so linux users utilise curl.